source/renderer/zettlr-editor.js

```diff
@@ -124,6 +124,9 @@ class ZettlrEditor {
     // Caches the "left" style property during distraction free
     this._leftBeforeDistractionFree = ''
 
+    this._spaceWidth = 0
```

Comment:
Since we're caching the values, what would happen when a user changes the theme with the font?

Reply:
Good point. Didn't know that there were different themes. I now added code that listens to theme changes and resets the width. This seems to work somewhat.

For some reason, some of the value of the text-indent in the other themes are off (the padding seems to be ok). I tried to investigate but couldn't find the problem. Maybe this is something for another PR or do you have a suggestion?